src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp

```diff
@@ -154,34 +162,34 @@ namespace argos
       CVector3 cOrientationError(cOrientationTarget - m_cOrientation);
       /* desired  roll, pitch, yaw rates */
       CVector3 cAngularVelocityTarget =
-         (cOrientationTarget - m_cOrientationTargetPrev) / GetPM3DEngine().GetPhysicsClockTick();
+         (cOrientationTarget - m_cPrevOrientationTarget) / GetPM3DEngine().GetPhysicsClockTick();
```

Comment:
It is very difficult to see the important changes with all these name changes to variables...